keyboards/rgbkb/sol/keymaps/xyverz/keymap.c

```diff
@@ -0,0 +1,366 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
```

Comment:
`#include QMK_KEYBOARD_H` includes `bootloader.h` automatically, so this line is redundant.

```suggestion
```

Reply:
Good to know.